### PR TITLE
Task/f2fin/switched to web

### PR DESCRIPTION
**Pull request by adusmani17**

### CODE REVIEW

File: src/components/chatbot/chatbot.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          messageList={this.state.messageList}
+          showEmoji
+        />
+      </div>
+    );
+  }
+}
+export default Chatbot;
+
+// import { Box, Container, Typography } from "@mui/material";
```
Comment (coder-faraz) [inline src/components/chatbot/chatbot.jsx:52]: dekhkar to push kia karo code.......why this commented code?

**Review by coder-faraz — CHANGES_REQUESTED** [review]
left comments

File: src/components/chatbot/chatbot.jsx (hunk 1 above)
Comment (adusmani17) [reply src/components/chatbot/chatbot.jsx:52]: This code will be implemented in future.

**Review by coder-faraz — APPROVED** [review]
looking good

**Pull request merged**